### DISCUSSION TITLE
how to import modules like navigation.

### DISCUSSION THREAD

**Discussion by izhar360** [nolimits4web/swiper#6841] [General]
The docs say import it like:

`import { Navigation, Pagination, Scrollbar, A11y } from 'swiper/modules';`

but it doesn't work, this works:

`import { Navigation, Pagination, Scrollbar, A11y } from 'swiper'`

**Comment by ckanitz** [comment]
Check your version. the import from `swiper/modules` was introduced in version 10.
If you use version 9 or below everything is imported from `swiper`

**Comment by dadoadk** [comment]
Use Elements

// import function to register Swiper custom elements
import { register } from 'swiper/element/bundle';
// register Swiper custom elements
register();
